public/components/trace_analytics/components/services/services_table.tsx

```diff
@@ -65,20 +68,15 @@ export function ServicesTable(props: ServicesTableProps) {
     isServiceTrendEnabled,
     setIsServiceTrendEnabled,
     serviceTrends,
+    dataSourceMDSId,
   } = props;
 
   const selectionValue = {
     onSelectionChange: (selections: any[]) => setSelectedItems(selections),
   };
 
   const nameColumnAction = (serviceName: string) => {
-    addFilter({
-      field: mode === 'jaeger' ? 'process.serviceName' : 'serviceName',
-      operator: 'is',
-      value: serviceName,
-      inverted: false,
-      disabled: false,
-    });
+    window.location.href = generateServiceUrl(serviceName, dataSourceMDSId[0].id);
```

Comment:
The change looks good to me. Can you please add cypress tests and / or unit tests for this? Thanks!

Reply:
Added jest test 
it('redirects to the correct URL when the service link is clicked',